dotCMS/src/main/java/com/dotmarketing/portlets/fileassets/business/IFileAsset.java

```diff
@@ -68,7 +68,9 @@
 
 	public long getFileSize();
 
-	public void setFileName(String name);
+	public void setUnderlyingFileName(String name);
```

Comment:
please add good documentation to understand what is it

Reply:
all documentation is in the Impl FileAsset Class
That interface does not have a single comment on it.